### ISSUE TITLE
SCP-184 的附加故事《Alternate Character Interpretations》和《Making a Scene》尚未翻译

### ISSUE THREAD

**Issue by 7sDream** [7sDream/scp-pdf#16]
- [ ] 等待翻译[《Alternate Character Interpretations》](http://scp-wiki.wikidot.com/alternate-character-interpretations)
- [ ] 等待翻译[《Making a Scene》](http://scp-wiki.wikidot.com/make-a-scene)
- [ ] 更新所有附加文档到 PDF 中

SCP-184 中有超链

**Comment by ko30190** [comment]
What's up？You sure it is sent to me？



---原始邮件---
发件人: "七秒不觉梦"<notifications@github.com&gt;
发送时间: 2020年4月29日(星期三) 下午4:36
收件人: "7sDream/scp-pdf"<scp-pdf@noreply.github.com&gt;;
抄送: "Subscribed"<subscribed@noreply.github.com&gt;;
主题: [7sDream/scp-pdf] SCP-184 的附加故事《Alternate Character Interpretations》和《Making a Scene》尚未翻译 (#16)




  
 等待翻译《Alternate Character Interpretations》 

 等待翻译《Making a Scene》 

 更新所有附加文档到 PDF 中

**Comment by 7sDream** [comment]
No, I just open a issue in this repo and it's a memo for myself.

Why you received a notification email, have you recently subscribed to any topics?
